### ISSUE TITLE
Reorder items on main view

### ISSUE THREAD

**Issue by towebo** [openscape-community/openscape#23]
Depending on your use case the layout of the main view can be optimized.
The least interaction with the app is if you just start it and walk around and get info about what you're passing.

The next level of interaction is when you want to know where you are and what's around you.
The next level is when you want to lookup a place, place marker or want to save your current location as a place marker

My suggestion is that the actions are ordered depending on how much interactivity you put into your usage.

Suggestion:
Menu
Inactivate
Label: Listen to my surroundings
My Location
Place Markers Nearby
Around Me
In Front Of Me

Search Field: Select Destination
Places Nearby
Place Markers and Routs
Current Location

Preferable would be if the VoiceOver cursor could focus the "My Location" button when the app starts. I've done this in some of my apps but they are built with C# and UIKit.

**Comment by brunoprietog** [comment]
I feel that the button placement is quite good. I like that the most important ones are at the bottom, it's easier to reach them than if they are at the top, because you have to stretch your finger more

**Comment by steinbro** [comment]
I agree there's probably room to improve the views, but the existing layout gets a lot of weight, because of its consistency with and familiarity to Soundscape users. We're open to discussion, but you'd need to make a strong case for change.
